build-parent/pom.xml

```diff
@@ -259,6 +259,12 @@
         <version>${version.software.tnb}</version>
         <scope>test</scope>
       </dependency>
+      <dependency>
+        <groupId>software.tnb</groupId>
+        <artifactId>system-x-hyperfoil</artifactId>
+        <version>1.0-SNAPSHOT</version>
```

Comment:
(I guess this might be the reason for you to have this PR as draft)

I think we should use `${version.software.tnb}` here though having a look at the [TNB repo,](https://github.com/tnb-software/TNB)

Though it seems that now the `version.software.tnb` property should also be changed to `1.0-SNAPSHOT` instead of `0.0.2`. Have a look at https://github.com/tnb-software/TNB/commit/d380c78da65d4aa9c1032c28ff7a53173a5a215f